src/consensus/pbft_manager.cpp

```diff
@@ -1487,8 +1487,6 @@ void PbftManager::pushSyncedPbftBlocksIntoChain_() {
         break;
       }
 
-      syncBlockQueuePop();
```

Comment:
I remember Jakub mentioned that we cannot remove this.
For example, sync_queue_ has only one PBFT block period 11. If we pop the block from sync_queue_ immediately. And has not push into PBFT chain. That will cause sync the block period 11 again from peers and push into sync_queue_.

Reply:
those changes are part of https://github.com/Taraxa-project/taraxa-node/pull/1118